example/src/main/java/com/buglife/android/example/MainActivity.java

```diff
@@ -38,16 +35,10 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     public void reportBugButtonTapped(View view) {
-        try {
-            Bitmap screenshot = Buglife.getScreenshotBitmap();
-            File file = new File(getCacheDir(), "Screenshot.png");
-            screenshot.compress(Bitmap.CompressFormat.PNG, 100, new FileOutputStream(file));
-            FileAttachment attachment = new FileAttachment(file, MimeTypes.PNG);
-            Buglife.addAttachment(attachment);
-            Buglife.showReporter();
-        } catch (FileNotFoundException e) {
-            e.printStackTrace();
-        }
+        Bitmap screenshot = Buglife.getScreenshotBitmap();
+        Attachment attachment = new Attachment.Builder("Screenshot.png", TYPE_PNG).build(screenshot);
```

Comment:
Sorry, perhaps I wasn't clear here... The sample code _should_ be using the new (non-deprecated) API, however your initial PR illustrated that the new API results in significantly more code on the side of the API consumer. Any thoughts on how we can improve the FileAttachment API to make it easier to create a screenshot attachment?

Reply:
Oh, I see. I don't necessarily think the `FileAttachment` API should necessarily change. It's doing just about everything it should be doing. The reason the number of lines increased is due to the need to create a file from the captured bitmap. So, maybe the simplification should happen there. Currently, we have `Buglife#getScreenshotBitmap()`, but maybe we should deprecate that in favor of a `Buglife#captureScreenshot()` that returns a `FileAttachment` instead of a `Bitmap`. That would turn the sample code into something like:

```
FileAttachment screenshot = Buglife.captureScreenshot();
Buglife.addAttachment(screenshot);
Buglife.showReporter();
```

Reply:
Hmmm, you might be right. It's a bummer that there's more code involved for attaching files, although maybe that's okay (sorry, still not entirely used to the verbosity of Java 😁). And I do like your `captureScreenshot()` suggestion. 

@danhd123 any thoughts regarding the new `FileAttachment` API?